README.md

```diff
@@ -15,6 +15,10 @@ For Mac users, I highly recommend iTerm 2 + Solarized Dark
 
 **NOTE:** In all likelihood, you will need to install a [Powerline-patched font](https://github.com/Lokaltog/powerline-fonts) for this theme to render correctly.
 
+For iTerm 2 users, you have to set the font : iTerm 2 -> Preferences -> Profiles -> Text and use one of the Meslo fonts :
```

Comment:
There is no requirement about the "Meslo" font : any "*Powerline*" font can be used :-)

(in my case, using "Sauce Code Powerline")